.ci/component_descriptor

```diff
@@ -1,95 +1,26 @@
 #!/bin/bash
 
-# SPDX-FileCopyrightText: 2020 SAP SE or an SAP affiliate company and Gardener contributors
+# SPDX-FileCopyrightText: 2023 SAP SE or an SAP affiliate company and Gardener contributors
 #
 # SPDX-License-Identifier: Apache-2.0
 
 set -o errexit
 set -o nounset
 set -o pipefail
 
-echo "component-cli is required to generate the component descriptors"
-CLI_PATH="$(mktemp -d)"
-COMP_CLI=${CLI_PATH}/component-cli
-echo "Trying to installing component-cli to ${COMP_CLI}"
-OS="$(uname -o | awk '{print tolower($0)}')"
-ARCH="$(uname -m | awk '{print tolower($0)}')"
-
-if [[ ${OS} == *"linux"* ]]; then
-  OS="linux"
-fi
-if [[ ${OS} == *"darwin"* ]]; then
-  OS="darwin"
-fi
-
-if [[ ${ARCH} == *"x86_64"* ]]; then
-  ARCH="amd64"
-fi
-if [[ ${ARCH} == *"arm64"* ]]; then
-  ARCH="arm64"
-fi
-
-curl -L https://github.com/gardener/component-cli/releases/download/v0.48.0/componentcli-${OS}-${ARCH}.gz | gzip -d > ${COMP_CLI}
-chmod +x ${COMP_CLI}
+echo "create dummy component descriptor BEGIN"
 
 SOURCE_PATH="$(dirname $0)/.."
 VERSION="$(${SOURCE_PATH}/hack/get-version.sh)"
-COMMIT_SHA="$(git --git-dir ${SOURCE_PATH}/.git rev-parse HEAD)"
-
-printf "> Building components with version ${VERSION} - ${COMMIT_SHA}\n"
-
-REPO_CTX="${CURRENT_COMPONENT_REPOSITORY}"
-
-# creates a component archive for deployer
-# it expects 1 argument with
-# $1 is the name of the component
-function buildComponentArchive() {
-  COMPONENT_NAME=$1
-  CA_PATH="$(mktemp -d)"
-  printf "> Building component ${COMPONENT_NAME}\n"
-
-  COMPONENT_REFERENCES=""
-
-  if [ -f ${SOURCE_PATH}/.landscaper/${COMPONENT_NAME}/component-references.yaml ]; then
-    COMPONENT_REFERENCES="-c ${SOURCE_PATH}/.landscaper/${COMPONENT_NAME}/component-references.yaml"
-  fi
-
-  ${COMP_CLI} ca "${CA_PATH}" "${CTF_PATH}" \
-    --component-name=github.com/gardener/landscaper/${COMPONENT_NAME} \
-    --component-version=${VERSION} \
-    --repo-ctx=${REPO_CTX} \
-    -s ${SOURCE_PATH}/.landscaper/sources.yaml \
-    -r ${SOURCE_PATH}/.landscaper/${COMPONENT_NAME}/resources.yaml \
-    COMMIT_SHA=${COMMIT_SHA} \
-    VERSION=${VERSION} \
-    $COMPONENT_REFERENCES
-}
-
-buildComponentArchive "container-deployer"
-buildComponentArchive "helm-deployer"
-buildComponentArchive "manifest-deployer"
-buildComponentArchive "mock-deployer"
-
-# add landscaper component descriptor
-printf "> Create Landscaper ca archive\n"
-LS_CA_PATH="$(mktemp -d)"
-cp ${BASE_DEFINITION_PATH} "${LS_CA_PATH}/component-descriptor.yaml"
+REPO_CTX="eu.gcr.io/gardener-project/development"
```

Comment:
ideally this shouldn't be hardcoded.
The base-url is currently passed along as `CURRENT_COMPONENT_REPOSITORY` in the component-descriptor script, but if you plan to reuse that later that's also not really an option.

Reply:
I rather not rely on implicit environment variables when this script is meant to run indipendently (and derive from standard CI/CD logic) from the CI/CD versions. The dev repo was already hardcoded for a long time.

In the future I would rather like to discuss how CI/CD tooling can support OCM transport formats created with OCM cli instead of expecting a plain component descriptor yaml file.